src/languages/en-US.ts

```diff
@@ -1466,6 +1474,28 @@ export default class extends Language {
 			examples: ['@Pete Attempted to mention everyone.']
 		}),
 
+		/**
+		 * ##################
+		 * POKEMON COMMANDS
+		 */
+		COMMAND_POKEDEX_EMBED_DATA: {
+			types: 'Type(s)',
+			abilities: 'Abilities',
+			genderRatio: 'Gender Ratio',
+			smogonTier: 'Smogon Tier',
+			unknownSmogonTier: 'Unknown / Alt form',
+			height: 'Height',
+			weight: 'Width',
+			eggGroups: 'Egg group(s)',
+			otherFormes: 'Other forme(s)',
+			evolutionaryLine: 'Evolutionary line',
+			baseStats: 'Base stats',
+			flavourText: 'Pokdex entry',
+			externalResources: 'External resources',
+			none: 'None'
```

Comment:
The convention is that subkeys should be in SCREAMING_CASE as well, likewise all other keys.